src/main/java/com/ibm/watson/apis/conversation_enhanced/rest/ProxyResource.java

```diff
@@ -53,8 +53,11 @@
   private static String PASSWORD;
```

Comment:
`USERNAME` and `PASSWORD` variables should not be static and should not be actually variables. You only need them when you create the service instance. 

`public ProxyResource()` should be:

```java
public ConversationInterceptor() {
  ConversationService service = new ConversationService(ConversationService.VERSION_DATE_2016_05_19);
  service.setUsernameAndPassword(System.getenv("CONVERSATION_USERNAME"), System.getenv("CONVERSATION_PASSWORD"));
  if (URL != null) {
    service.setEndPoint(URL);
  }
}
```